rust/src/quic/quic.rs

```diff
@@ -587,6 +614,13 @@ pub unsafe extern "C" fn SCRegisterQuicParser() {
         ALPROTO_QUIC = alproto;
         if SCAppLayerParserConfParserEnabled(ip_proto_str.as_ptr(), parser.name) != 0 {
             let _ = AppLayerRegisterParser(&parser, alproto);
+            if let Some(val) = conf_get("app-layer.protocols.quic.encryption-handling") {
+                if let Ok(eh) = EncryptionHandling::from_str(val) {
+                    unsafe { ENCRYPTION_BYPASS_ENABLED = eh };
+                } else {
+                    SCLogError!("Unknown value {} for quic.encryption-handling.", val);
```

Comment:
This still allows the bad setting, it now prints and error but acts like a warning. I think we should just return error or fatal error.